### PR TITLE
Updating to point everyone to docs tutorials

### PR DESCRIPTION
**Pull request by moxiegirl**
In this PR closes #683 

- removed tutorials directory
- created redirects to new docs.blockstack.org
- Updated a couple of FAQs 

The old site had a "storage tutorial" that went to an old employees personal script playground.  It did not have any actual detailed GAIA code. The new multi player tutorial covers all the GAIA material and the bits of that old storage one that seemed somewhat useful.

I tested this locally but it might be smart pulling this pr and checking it out yourself just in case.  The other thing is, it would be great if we could get this PR merged and pushed before the 24th.

Signed-off-by: Mary Anthony <mary@blockstack.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
@aulneau I closed that other PR to avoid a complex merge. I didn't want to do a bad merge and potentially break something.  Here are my changes with your comments incorporated. They are passing cheks.

File: components/top-area/TopArea.js [hunk 1]
```diff
@@ -49,8 +49,7 @@ const TopArea = (props) => {
           <div>
             <div className="grid-flex v-spaced break-xlg">
               <div className="col-4">
-                <Link prefetch href="/tutorials/hello-blockstack">
-                  <a className="g-card code-card dark align-left link">
+                  <a className="g-card code-card dark align-left link" href="https://docs.blockstack.org/browser/hello-blockstack.html">
```
Comment (aulneau) [inline components/top-area/TopArea.js]:
```suggestion
                  <a className="g-card code-card dark align-left link" target="_blank" href="https://docs.blockstack.org/browser/hello-blockstack.html">
```
We should have these open in new tabs/windows

File: components/top-area/TopArea.js [hunk 2]
```diff
@@ -80,11 +79,9 @@ const TopArea = (props) => {
                       </div>
                     </Card>
                   </a>
-                </Link>
               </div>
               <div className="col-4">
-                <Link prefetch href="/tutorials/todo-list">
-                  <a className="g-card code-card dark align-left link">
+                  <a className="g-card code-card dark align-left link" href="https://docs.blockstack.org/browser/todo-list.html">
```
Comment (aulneau) [inline components/top-area/TopArea.js]:
```suggestion
                  <a className="g-card code-card dark align-left link" target="_blank" href="https://docs.blockstack.org/browser/todo-list.html">
```

File: components/top-area/TopArea.js [hunk 3]
```diff
@@ -111,11 +108,9 @@ const TopArea = (props) => {
                       </div>
                     </Card>
                   </a>
-                </Link>
               </div>
               <div className="col-4">
-                <Link href="/tutorials/multi-player-storage">
-                  <a className="g-card code-card dark align-left link">
+                  <a className="g-card code-card dark align-left link" href="https://docs.blockstack.org/browser/blockstack_storage.html">
```
Comment (aulneau) [inline components/top-area/TopArea.js]:
```suggestion
                  <a className="g-card code-card dark align-left link"  target="_blank" href="https://docs.blockstack.org/browser/blockstack_storage.html">
```

File: components/top-area/TopArea.js (hunk 1 above)
Comment (moxiegirl) [reply components/top-area/TopArea.js]: Sure. I'll add that.

**Comment by moxiegirl** [comment]
@aulneau incorporated and tested.

**Pull request merged**